src/main/java/org/jabref/gui/frame/JabRefFrame.java

```diff
@@ -171,10 +183,6 @@ public JabRefFrame(Stage mainStage,
                 this.preferences,
                 taskExecutor);
 
-        this.fileHistory = new FileHistoryMenu(
-                this.preferences.getLastFilesOpenedPreferences().getFileHistory(),
-                dialogService,
-                getOpenDatabaseAction());
         this.setOnKeyTyped(key -> {
             if (this.fileHistory.isShowing()) {
```

Comment:
Why wasn't this moved up? All functionality of the fileHistory menu should be kept togeher.